arangod/Aql/EngineInfoContainerDBServerServerBased.h

```diff
@@ -56,6 +56,9 @@ class QuerySnippet;
 
 class EngineInfoContainerDBServerServerBased {
  private:
+  // TODO Temporary. Do not check in!!
```

Comment:
Open / outdated todo?

Reply:
Keep, we need to remove this on BaseBranch -> Devel merge